portafly/src/App.tsx

```diff
@@ -0,0 +1,90 @@
+import 'react-app-polyfill/ie11'
+import '@patternfly/react-core/dist/styles/base.css'
+import { Brand } from '@patternfly/react-core'
+import React from 'react'
+import {BrowserRouter as Router, Redirect, useHistory} from 'react-router-dom'
```

Comment:
We need to add a linter ASAP and the more opinionated the better. I can do it.

Reply:
https://issues.redhat.com/browse/THREESCALE-4615